### PR TITLE
N31: Blackjack

### PR DESCRIPTION
**Pull request by setzer22**
Here's my writeup for Blackjack :smile: 

_Part of #949_

### CODE REVIEW

**Review by 17cupsofcoffee — APPROVED** [review]

File: content/news/031/index.md [hunk 1]
```diff
@@ -74,6 +74,33 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+source release, several new features have been added:
+
+- Added a resizeable viewport system, with node graph pan and zoom.
+- Built an initial implementation for a properties inspector and geometry
+  spreadsheet panels.
+- Added a subdivision node, with a fast catmull-clark subdivision technique
+ based on
+ [this recent paper](https://onrendering.com/data/papers/catmark/HalfedgeCatmullClark.pdf)
+- Separated the node graph functionality into
+ [its own crate](https://github.com/setzer22/egui_node_graph)
```
Comment (AngelOnFira) [inline content/news/031/index.md:96]: Please move all links to the bottom section :)

File: content/news/031/index.md [hunk 2]
```diff
@@ -74,6 +74,33 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Tooling Updates
 
+### [Blackjack]
+
+![Blackjack: Showcase of the new catmull-clark subdivision](blackjack.gif)
+
+[Blackjack] by @setzer22 is a new procedural modeling application made in Rust,
```
Comment (AngelOnFira) [inline content/news/031/index.md:81]: Just want to check, but do you want to add any links to your username? Such as Twitter or Github?

**Review by AngelOnFira — CHANGES_REQUESTED** [review]

**Review by AngelOnFira — APPROVED** [review]

File: content/news/031/index.md (hunk 1 above)
Comment (setzer22) [reply content/news/031/index.md]: Oops, sorry I missed the message! Will keep that in mind for the next time

File: content/news/031/index.md (hunk 2 above)
Comment (setzer22) [reply content/news/031/index.md:236]: It's fine as-is :+1: I don't have a twitter with my username and I'm already sharing links to my github projects so people can easily find me there

**Pull request merged**